collab-folder/src/core/view.rs

```diff
@@ -23,6 +25,7 @@ const VIEW_LAYOUT: &str = "layout";
 const VIEW_CREATE_AT: &str = "created_at";
 const VIEW_ICON_URL: &str = "icon_url";
 const VIEW_COVER_URL: &str = "cover_url";
+const FAVORITE_STATUS: &str = "is_favorite";
```

Comment:
It seems unnecessary to add FAVORITE_STATUS because we can retrieve the favorite view from the FavoritesArray.

Reply:
Yeah that makes sense, I kept it there with a thought process that we might want to figure out whether a view is favorited without having to iterate through the array but still I guess it would have its own downsides I'll remove it.